### ISSUE TITLE
RM 2.0.6 cannot open ...rscons-ranges.tsv for reading

### ISSUE THREAD

**Issue by rattei** [Dfam-consortium/RepeatModeler#264]
Hello,

we have installed RM 2.0.6 on a RHEL9.5 Linux system, on which RM 2.0.5 worked well. Dependencies are the same for RM 2.0.5 and 2.0.6:
Search Engine = rmblast 2.14.1+
Dependencies: TRF 4.09, RECON , RepeatScout 1.0.6, RepeatMasker 4.1.7
LTR Structural Analysis: Enabled ( GenomeTools 1.6.2, LTR_Retriever v3.0.1,
                                   Ninja /tmp/test/yeast288c, MAFFT 7.526,
                                   CD-HIT 4.8.1 )

The example run works with 2.0.5 but fails with 2.0.6.

How to reproduce:
```
mkdir /tmp/test
cd /tmp/test
wget https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/146/045/GCF_000146045.2_R64/GCF_000146045.2_R64_genomic.fna.gz
gunzip GCF_000146045.2_R64_genomic.fna.gz
BuildDatabase -name yeast288c GCF_000146045.2_R64_genomic.fna
RepeatModeler -threads 4 -LTRStruct  -database yeast288c

End of output and error message: 

...
RepeatModeler Round # 1
========================
Searching for Repeats
 -- Sampling from the database...
   - Gathering up to 40000000 bp
   - Final Sample Size = 12157082 bp ( 12157082 non ambiguous )
   - Num Contigs Represented = 17
   - Sequence extraction : 00:00:02 (hh:mm:ss) Elapsed Time
 -- Running RepeatScout on the sequences...
   - RepeatScout: Running build_lmer_table ( l = 13 )..
   - RepeatScout: Running RepeatScout.. : 137 raw families identified
   - RepeatScout: Running filtering stage.. 127 families remaining
   - RepeatScout: 00:03:42 (hh:mm:ss) Elapsed Time
   - Large Satellite Filtering.. : 1 found in 00:00:03 (hh:mm:ss) Elapsed Time
Could not open /tmp/test/RM_731820.WedNov270641122024/round-1/sampleDB-1.fa.rscons-ranges.tsv for reading!
```
best,
Thomas

**Comment by rmhubley** [comment]
That is because RepeatModeler 2.0.6 has RepeatScout 1.0.7 as a prerequisite.  I am surprised that the configure script didn't prompt for this version specifically.

**Comment by rattei** [comment]
I can confirm that repeatmodeler 2.0.6 runs the example without errors, if configured with repeatscout 1.0.7.

We would very much appreciate informative error messages, as well as dependency checks by the installation script.

We would also very much appreciate if the repeatmasker.org and github.com  websites would give consistent information about dependencies (currently the statements about prerequisites are contradicting - see my comment above).

**Comment by rmhubley** [comment]
Sorry that you had conflicting information, but I believe you have a cached browser page for "https://www.repeatmasker.org/RepeatModeler/".  If you refresh that page you will see it was removed at the time of release -- I am not sure why your browser didn't automatically detect that.  On the front page of the RepeatMasker website the RepeatModeler link is pointing to the github repository.  When we released 2.0.6 we decided to no longer maintain two different help pages on RepeatModeler and instead default to the github site for that type of information.
